isis/src/base/apps/hist/tsts/default/Makefile

```diff
@@ -1,9 +1,20 @@
+# 2020-06-25 - Kristin Berry - Added a test to check that if nbins is set, it is used.
+#   To make this test, I set the number of bins to much smaller number than the internal default,
+#   and just checked that the output was different from the default test. 
```

Comment:
I think this is me not being good at reading Cpp, but where does the comparison to the default test happen? I see both of the results going to dev/null

Reply:
You're right, it's not doing this comparison! I'll update this comment. (My intent behind the comment was to explain that I didn't _really_ verify the truthdata for the test I added. I just verified that it was _different_ than the truthdata without `nbins` set.)